writeengine/server/we_dmlcommandproc.cpp

```diff
@@ -589,11 +590,11 @@ uint8_t WE_DMLCommandProc::processSingleInsert(messageqcpp::ByteStream& bs, std:
             if (mapIter != m_txnLBIDMap.end())
             {
                 SP_TxnLBIDRec_t spTxnLBIDRec = (*mapIter).second;
-                std::tr1::unordered_map<BRM::LBID_t, uint32_t> ::iterator listIter = spTxnLBIDRec->m_LBIDMap.begin();
+                std::vector<BRM::LBID_t>::iterator listIter = spTxnLBIDRec->m_LBIDs.begin();
```

Comment:
I believe we can replace this long type definition with auto thx to c++11 :)

Reply:
Will do.